XenModel/Actions/VM/CreateVMAction.cs

```diff
@@ -210,6 +210,7 @@ protected override void Run()
             AddDisks();
             AddNetworks();
             XenAdminConfigManager.Provider.ShowObject(VM.opaque_ref);
+            VM.IsBeingCreated = false;

```

Comment:
There is another action, CreateVMFastAction, that is using the same mechanism of hiding the VM while it's being created. We should set the IsBeingCreated property there as well.